### ISSUE TITLE
ENH: Make files read-only on finalize

### ISSUE THREAD

**Issue by Eric89GXL** [andreyzhd/VideoMEG#6]
I haven't checked to see what the state of the output files are, but if they are not marked as read-only after they are closed, perhaps they should be (like the original `.fif` files are). Okay if I implement this if it's not already done?

**Comment by andreyzhd** [comment]
Right now they are not. This would be a nice feature, even though we never had any problems with the files not being read-only.

**Comment by Eric89GXL** [comment]
Yeah it's more of just a user-safety / dummy-proofing thing, so I'm not surprised you haven't needed it yet :)

**Comment by Eric89GXL** [comment]
Changes made in #4.
